opts/network.go

```diff
@@ -0,0 +1,101 @@
+package opts
+
+import (
+	"encoding/csv"
+	"fmt"
+	"strings"
+
+	"github.com/docker/docker/api/types/network"
+)
+
+const (
+	networkOptName  = "name"
+	networkOptAlias = "alias"
+	driverOpt       = "driver-opt"
+)
+
+// NetworkOpt represents a network config in swarm mode.
+type NetworkOpt struct {
+	options []network.Options
+}
+
+// Set networkopts value
+func (n *NetworkOpt) Set(value string) error {
+	csvReader := csv.NewReader(strings.NewReader(value))
+	fields, err := csvReader.Read()
+	if err != nil {
+		return err
+	}
+	var netOpt network.Options
+
+	if len(fields) == 1 && !strings.Contains(fields[0], "=") {
```

Comment:
Can you instead use the common pattern used in other similar options such as `opts/port.go`.

```
longSyntax, err := regexp.MatchString(`\w+=\w+(,\w+=\w+)*`, value)
```